client/productOrder.jsx

```diff
@@ -5,51 +5,127 @@ class ProductOrder extends React.Component {
+
     return (
+
       <div>
+
         <div>
           LIMITED EDITION: {this.state.selectedColorway}
         </div>
         <div id="outer_colorway_holder" style={{display: "inline-block"}}>
           <div id="upper_colorway_holder" style={{display: "inline-block"}}>
-            <StatefulButton text="Thunder (Dark Blue Upper/White Sole" style={{display: "inline-block"}}/>
-            <StatefulButton text="Geyser (Light Blue Sole)" style={{display: "inline-block"}}/>
-            <StatefulButton text="Quartz (Light Grey Sole" style={{display: "inline-block"}}/>
-            <StatefulButton text="Obsidian (Black Sole" style={{display: "inline-block"}}/>
-            <StatefulButton text="Flame (Light Red Sole)" style={{display: "inline-block"}}/>
-            <StatefulButton text="Bluster (Grey Sole)" style={{display: "inline-block"}}/>
+            <StatefulButton text="Thunder (Dark Blue Upper/White Sole" style={{display: "inline-block", padding: "3px"}}/>
+            <StatefulButton text="Geyser (Light Blue Sole)" style={{display: "inline-block", padding: "3px"}}/>
+            <StatefulButton text="Quartz (Light Grey Sole" style={{display: "inline-block", padding: "3px"}}/>
+            <StatefulButton text="Obsidian (Black Sole" style={{display: "inline-block", padding: "3px"}}/>
+            <StatefulButton text="Flame (Light Red Sole)" style={{display: "inline-block", padding: "3px"}}/>
+            <StatefulButton text="Bluster (Grey Sole)" style={{display: "inline-block", padding: "3px"}}/>
           </div>
           <div id="lower_colorway_holder">
-            <StatefulButton text="Blizzard (White Sole" style={{display: "inline-block"}}/>
-            <StatefulButton text="Cyclone (Light Green Sole)" style={{display: "inline-block"}}/>
-            <StatefulButton text="Pukeko (White Sole)" style={{display: "inline-block"}}/>
+            <StatefulButton text="Blizzard (White Sole" style={{display: "inline-block", padding: "3px"}}/>
+            <StatefulButton text="Cyclone (Light Green Sole)" style={{display: "inline-block", padding: "3px"}}/>
+            <StatefulButton text="Pukeko (White Sole)" style={{display: "inline-block", padding: "3px"}}/>
           </div>
         </div>
         <div id="outer_size_holder">
           <div id="upper_size_holder">
-            <StatefulButton text="8" style={{display: "inline-block"}}/>
-            <StatefulButton text="8.5" style={{display: "inline-block"}}/>
-            <StatefulButton text="9" style={{display: "inline-block"}}/>
-            <StatefulButton text="9.5" style={{display: "inline-block"}}/>
-            <StatefulButton text="10" style={{display: "inline-block"}}/>
-            <StatefulButton text="10.5" style={{display: "inline-block"}}/>
-            <StatefulButton text="11" style={{display: "inline-block"}}/>
+            <StatefulButton style={{display: "inline-block", padding: "3px"}} image={button8}/>
```

Comment:
Would a button class with display:inline block and padding: "3px" be useful here?